ui/test/parseError.test.ts

```diff
@@ -639,4 +639,30 @@ You do not have permission to view this directory or page using the credentials
         assert.strictEqual(pe.message, 'Incorrect authentication credentials.');
         assert.strictEqual(pe.isUserCancelledError, false);
     });
+
+    test('XML errors', () => {
+        const err1: {} = {
+            statusCode: 409,
+            message: `<?xml version="1.0" encoding="utf-8"?><Error><Code>ContainerBeingDeleted</Code><Message>The specified container is being deleted. Try operation later.
+RequestId:35eae91c-a01e-0010-1b5c-ca2979000000
```

Comment:
The previous release also displayed requestId and time. I think RequestId is helpful if the user wants to talk to Azure Support. Also it just makes sense to me to display the whole "Message" property